lib/trento_web/views/v1/sap_system_view.ex

```diff
@@ -119,6 +120,8 @@ defmodule TrentoWeb.V1.SapSystemView do
     |> Map.from_struct()
     |> Map.delete(:__meta__)
     |> Map.delete(:deregistered_at)
+    |> Map.delete(:database)
+    |> Map.delete(:database_id)
```

Comment:
Let's keep the `database_id`. I will be used in the frontend.
Or we can put it back in other PR, and work on the openapi schema, and so on.
Both are fine

Reply:
Yeah, don't change this. Let's leave it for other PR, so we do the openapi work there